integration/makefile.mk

```diff
@@ -0,0 +1,50 @@
+GOFAIL_BINARY = $(shell pwd)/gofail
+
+.PHONY: run-all-integration-tests
+run-all-integration-tests:
+	# we enable all failpoints
+	make gofail-enable
+
+	# we compile and execute all intergration tests
+	# add new integration test targets here
+	make run-integration-test-sleep 
+
+	# we disable all failpoints
+	make gofail-disable
+	make clean-gofail
```

Comment:
```suggestion
	$(MAKE) gofail-enable

	# we compile and execute all integration tests
	# add new integration test targets here
	$(MAKE) run-integration-test-sleep 

	# we disable all failpoints
	$(MAKE) gofail-disable
	$(MAKE) clean-gofail
```

Pls read https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html

Reply:
Amended the entire file with `$(MAKE)`